index.js

```diff
@@ -309,7 +309,7 @@ module.exports = class Filter extends Plugin {
         result = await this.processAndCacheFile(srcDir, destDir, entry, forceInvalidation, isChange, stats);
       } else {
         stats.linked++;
-        if (isChange) {
+        if (isChange && fs.existsSync(outputPath)) {
```

Comment:
I am not sure how did we end up in this state. We are expected to have the output path to be present if isChange is true. 
As far as I know, `isChange` is true only in the case of a rebuild, so `outputPath` must be present. I am not sure If I am missing something here.